ezbake/config/conf.d/auth.conf

```diff
@@ -1,39 +1,116 @@
 authorization: {
```

Comment:
It might be a good idea to include some meaningful documentation at the top of this file, like the old one has: https://github.com/puppetlabs/puppet/blob/master/conf/auth.conf#L1-L57

Rather than copy/paste the docs or write special new ones, we might be able to get away with just linking to the official published docs site (which we haven't published yet)?

@nfagerlund got any suggestions here?